docs/building-a-bento-amp-extension.md

```diff
@@ -551,23 +590,23 @@ exports.extensionBundles = [
 ];
 ```
 
-Note, if you are providing a version upgrade (pre-existing 0.1 to Bento 1.0, for example), it is important to note that the latest version is the 0.1 until the 1.0 version is no longer experimental and fully launched. The following is the most straight-forward way to specify this:
+The entry for your component must have `options.wrapper = "bento"`, and may optionally include `options.npm = true` to opt the Preact component portion for automated npm publishing on AMP's release schedule.
```

Comment:
```suggestion
The entry for your component must have `options.wrapper = "bento"`. It may optionally include `options.npm = true` to publish the Preact component portion on npm automatically, following AMP's release schedule.
```